app/src/main.ts

```diff
@@ -159,6 +165,19 @@ if (shouldQuit) {
         });
       });
     }
+    if (!appArgs.disableOldNativefierWarningyesiknowitisinsecure) {
+      if (
+        Math.floor(new Date().getTime() / 1000) - appArgs.buildDate >
+        5184000
```

Comment:
Please extract to a readable and maintainable multiplication, associated to an explicit constant name in all caps, ending with the unit:

```ts
const OLD_BUILD_WARNING_THRESHOLD_DAYS = 60;
const OLD_BUILD_WARNING_THRESHOLD_MS = const OLD_BUILD_WARNING_THRESHOLD_DAYS * 24 * 60 * 60 * 1000;
```

, then do your comparison